sdk/machinelearningservices/Azure.ResourceManager.MachineLearning/src/Azure.ResourceManager.MachineLearning.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Version>1.2.0-beta.2</Version>
+    <Version>2.0.0-alpha.1</Version>
```

Comment:
we have a principal that dotnet mgmt plane SDK cannot bump the major version, even if there are breaking changes on the service side without special exceptions.
Bumping the major version for one RP will require all the mgmt libraries to bump to the same version even if others do not have breaking changes.